threema-connector-webtest/src_test/threema/connector/webtest/MessageMultipleRecipientsTest.java

```diff
@@ -0,0 +1,52 @@
+package threema.connector.webtest;
+
+import static com.codeborne.selenide.Condition.empty;
+import static com.codeborne.selenide.Condition.selected;
+import static com.codeborne.selenide.Condition.value;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+import static com.codeborne.selenide.Selenide.open;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+
+import com.axonivy.ivy.webtest.IvyWebTest;
+import com.axonivy.ivy.webtest.engine.EngineUrl;
+import com.codeborne.selenide.ElementsCollection;
+
+@IvyWebTest(headless = false)
```

Comment:
headless = true or omit that annoation value .... otherwise the test may not run in headless environments (build-nodes)

Reply:
removed